tools/bcz.cmd

```diff
@@ -76,13 +76,46 @@ set _BUILD_CMDLINE="%MSBUILD%" %OPENCON%\OpenConsole.sln /t:"%_MSBUILD_TARGET%"
 
 echo %_BUILD_CMDLINE%
 echo Starting build...
+
+@rem start indeterminate progress in the taskbar
+@rem this `<NUL set /p =` magic will output the text _without a newline_
+<NUL set /p =]9;4;3;0

```

Comment:
nit: `9;4;3` will do! The `0` is not needed since we ignore the progress value in the indeterminate state (also it will automatically be set to 0 if not provided)